### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by SeraphDevelper** [JamesIves/fetch-api-data-action#716] [Q&A]
I try install enviroment for contributing but i get the next issue TypeScript. What version use of typescript for fix this issue? Thanks

![image](https://user-images.githubusercontent.com/49100873/189500440-7492793e-29f5-4df7-a75c-d9b5dd497cf8.png)

**Comment by JamesIves** [comment]
The version of TypeScript I use is the one in the package.json file. Do you have it installed globally by chance?
